components/tekton/pipelines/run-testsuite-from-image/run-testsuite-from-image.yaml

```diff
@@ -21,13 +21,29 @@ spec:
       # yamllint enable rule:line-length
     - name: FEATURE
       default: "sriov"
+      description: |
+        Feature in managed cluster under test.
     - name: CLUSTER_NAME
+      description: |
+        Name of the managed cluster
     - name: NAMESPACE
+      description: |
+        Namespace of managed cluster in management cluster.
     - name: NOTIF_URL
+      description: |
+        Webhook URL to send MESSAGE.
     - name: MESSAGE
+      description: |
+        Message to send.
     - name: WORKER1
+      description: |
+        Worker 1 name under test.
     - name: WORKER2
+      description: |
+        Worker 2 name under test.
     - name: INTERFACES
+      description: |
+        Ports in WORKER1 and WORKER2.
```

Comment:
Maybe we should call it `interface` instead of `port`?

Reply:
```suggestion
        Two equivalent interface ports present in WORKER1 and WORKER2.
```